src/main/java/com/coveo/source/SourceClient.java

```diff
@@ -162,7 +176,7 @@ public HttpResponse<String> deleteDocument(String sourceId, String documentId, B
     }
 
     /**
-     * Manage batches of items in a push source. See [Manage Batches of Items in a Push Source](https://docs.coveo.com/en/90)
+     * Manage batches of items in a push source. See [Manage Batches of Items in a Push SourceClient](https://docs.coveo.com/en/90)
```

Comment:
Were the renames of `Source` -> `SourceClient` in this file intended?

Reply:
nop , it is IDE  mistake and mine that i missed to double check this one